src/app/space/add-space-overlay/add-space-overlay.component.ts

```diff
@@ -75,6 +78,9 @@ export class AddSpaceOverlayComponent implements OnInit {
       this.space = this.createTransientSpace();
     }
     this.space.attributes.name = this.space.name.replace(/ /g, '_');
+    if (this.description.nativeElement.value.trim() != '') {
```

Comment:
I think we should set the description for the backend and empty or spaces are okay.

Reply:
Yeah sure, I can make this PR more straight-forward and open an issue for entering an empty space description.